ledger-core/inc/core/async/Future.hpp

```diff
@@ -253,26 +253,6 @@ namespace ledger {
                 });
             };
 
-            void callback(const Context& context, const std::function<void(std::experimental::optional<T>, std::experimental::optional<api::Error>)> & cb) {
-                onComplete(context, [cb] (const Try<T>& result) {
-                    if (result.isSuccess()) {
-                        cb(result.toOption().toOptional(), Option<api::Error>().toOptional());
-                    } else {
-                        cb(Option<T>().toOptional(), Option<api::Error>(result.getFailure().toApiError()).toOptional());
-                    }
-                });
-            };
-
-            void callback(const Context& context, const std::function<void(T, std::experimental::optional<api::Error>)> & cb) {
-                onComplete(context, [cb] (const Try<T>& result) {
-                    if (result.isSuccess()) {
-                        cb(result.getValue(), Option<api::Error>().toOptional());
-                    } else {
-                        cb(nullptr, Option<api::Error>(result.getFailure().toApiError()).toOptional());
-                    }
-                });
-            };
-
```

Comment:
Where this piece of code is moved on ?

Reply:
Removed completely. We don’t use it anymore.